Sources/TokamakCore/Views/Navigation/ToolbarContentBuilder.swift

```diff
@@ -0,0 +1,452 @@
+    items = (v1, v2, v3, v4, v5, v6, v7)
+    _items = [
+      AnyView(v1),
+      AnyView(v2),
+      AnyView(v3),
+      AnyView(v4),
+      AnyView(v5),
+      AnyView(v6),
+      AnyView(v7),
+    ]
+  }
+
+  init<T1: View, T2: View, T3: View, T4: View, T5: View, T6: View, T7: View, T8: View>(
+    _ v1: ToolbarItem<ID, T1>,
+    _ v2: ToolbarItem<ID, T2>,
+    _ v3: ToolbarItem<ID, T3>,
+    _ v4: ToolbarItem<ID, T4>,
+    _ v5: ToolbarItem<ID, T5>,
+    _ v6: ToolbarItem<ID, T6>,
+    _ v7: ToolbarItem<ID, T7>,
+    _ v8: ToolbarItem<ID, T8>
+  ) where Items == (
+    ToolbarItem<ID, T1>,
+    ToolbarItem<ID, T2>,
+    ToolbarItem<ID, T3>,
+    ToolbarItem<ID, T4>,
+    ToolbarItem<ID, T5>,
+    ToolbarItem<ID, T6>,
+    ToolbarItem<ID, T7>,
+    ToolbarItem<ID, T8>
+  ) {
+    items = (v1, v2, v3, v4, v5, v6, v7, v8)
+    _items = [
+      AnyView(v1),
+      AnyView(v2),
+      AnyView(v3),
+      AnyView(v4),
+      AnyView(v5),
+      AnyView(v6),
+      AnyView(v7),
+      AnyView(v8),
+    ]
+  }
+
+  init<T1: View, T2: View, T3: View, T4: View, T5: View, T6: View, T7: View, T8: View, T9: View>(
+    _ v1: ToolbarItem<ID, T1>,
+    _ v2: ToolbarItem<ID, T2>,
+    _ v3: ToolbarItem<ID, T3>,
+    _ v4: ToolbarItem<ID, T4>,
+    _ v5: ToolbarItem<ID, T5>,
+    _ v6: ToolbarItem<ID, T6>,
+    _ v7: ToolbarItem<ID, T7>,
+    _ v8: ToolbarItem<ID, T8>,
+    _ v9: ToolbarItem<ID, T9>
+  ) where Items == (
+    ToolbarItem<ID, T1>,
+    ToolbarItem<ID, T2>,
+    ToolbarItem<ID, T3>,
+    ToolbarItem<ID, T4>,
+    ToolbarItem<ID, T5>,
+    ToolbarItem<ID, T6>,
+    ToolbarItem<ID, T7>,
+    ToolbarItem<ID, T8>,
+    ToolbarItem<ID, T9>
+  ) {
+    items = (v1, v2, v3, v4, v5, v6, v7, v8, v9)
+    _items = [
+      AnyView(v1),
+      AnyView(v2),
+      AnyView(v3),
+      AnyView(v4),
+      AnyView(v5),
+      AnyView(v6),
+      AnyView(v7),
+      AnyView(v8),
+      AnyView(v9),
+    ]
+  }
+
+  init<
+    T1: View,
+    T2: View,
+    T3: View,
+    T4: View,
+    T5: View,
+    T6: View,
+    T7: View,
+    T8: View,
+    T9: View,
+    T10: View
+  >(
+    _ v1: ToolbarItem<ID, T1>,
+    _ v2: ToolbarItem<ID, T2>,
+    _ v3: ToolbarItem<ID, T3>,
+    _ v4: ToolbarItem<ID, T4>,
+    _ v5: ToolbarItem<ID, T5>,
+    _ v6: ToolbarItem<ID, T6>,
+    _ v7: ToolbarItem<ID, T7>,
+    _ v8: ToolbarItem<ID, T8>,
+    _ v9: ToolbarItem<ID, T9>,
+    _ v10: ToolbarItem<ID, T10>
+  ) where Items == (
+    ToolbarItem<ID, T1>,
+    ToolbarItem<ID, T2>,
+    ToolbarItem<ID, T3>,
+    ToolbarItem<ID, T4>,
+    ToolbarItem<ID, T5>,
+    ToolbarItem<ID, T6>,
+    ToolbarItem<ID, T7>,
+    ToolbarItem<ID, T8>,
+    ToolbarItem<ID, T9>,
+    ToolbarItem<ID, T10>
+  ) {
+    items = (v1, v2, v3, v4, v5, v6, v7, v8, v9, v10)
+    _items = [
+      AnyView(v1),
+      AnyView(v2),
+      AnyView(v3),
+      AnyView(v4),
+      AnyView(v5),
+      AnyView(v6),
+      AnyView(v7),
+      AnyView(v8),
+      AnyView(v9),
+      AnyView(v10),
+    ]
+  }
+}
```

Comment:
<!--
  1 failure:  File should conta...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/Navigation/ToolbarContentBuilder.swift;
  Line: 452;
-->
- :no_entry_sign: File should contain 400 lines or less: currently contains 452 (`file_length`)